### PR TITLE
Implement `setFileTimes`

### PR DESCRIPTION
**Pull request by alexbiehl**
I need `touchFile` which uses `setFileTimes`. The implementation is based on the one in the `turtle` library. The conversion from `EpochTime` to `FILETIME` is done as described in https://support.microsoft.com/en-us/kb/167296.

### CODE REVIEW

File: src/System/PosixCompat/Files.hsc [hunk 1]
```diff
@@ -132,6 +134,7 @@ import System.IO (IOMode(..), openFile, hFileSize, hSetFileSize, hClose)
 import System.IO.Error
 import System.PosixCompat.Types
 import System.Win32.File hiding (getFileType)
+import System.Win32.Time
```
Comment (jystic) [inline src/System/PosixCompat/Files.hsc]: I know some of the other imports are not explicit, but would you mind being explicit here? I don't want to make things any worse than they already are.

**Comment by jystic** [comment]
This is great, thanks for the patch 👍 I'll merge it and make a release as soon as that import is fixed.

I don't think this needs a major version bump as it's unlikely anyone was relying on that function throwing an exception 😂

File: src/System/PosixCompat/Files.hsc (hunk 1 above)
Comment (alexbiehl) [inline src/System/PosixCompat/Files.hsc]: Done. I am short on time I will make the `System.Win32.File` imports explicit another time :)

**Comment by jystic** [comment]
Perfect, thanks! 🚗

**Comment by jystic** [comment]
Available as [unix-compat-0.4.3.0](http://hackage.haskell.org/package/unix-compat-0.4.3.0)

**Pull request merged**